src/TextArea/index.js

```diff
@@ -250,6 +250,8 @@ class Textarea extends React.Component {
       readonly,
       theme,
     } = this.props
+
+    const { ...otherProps } = this.props
```

Comment:
Generally, the `...otherProps` variable only grabs the props that haven't already been declared/used (in other words, it would sit between lines 251 and 252 in this file - see `InputText` for reference).  That way, the ones we don't want passed directly can be filtered, overridden, used elsewhere, etc.  I'm not sure it would have many practically different implications, particularly in this case, but that has generally been the pattern, and probably one we should either continue to follow or discuss changing elsewhere, as well?